assets/src/blocks/CarouselHeader/migrate.js

```diff
@@ -0,0 +1,13 @@
+const React = require('react');
+const ReactDOMServer = require('react-dom/server');
+
+const { CarouselHeaderStaticContent } = require('./CarouselHeaderStaticContent');
+
+const attributes = JSON.parse(process.argv[2]);
+
+const renderedCarousel = ReactDOMServer.renderToString(React.createElement(
+  CarouselHeaderStaticContent,
+  attributes
+));
+
+console.log(renderedCarousel);
```

Comment:
Strictly, I guess this should be `process.stdout.write` but for the PoC purpose `console` is ok, we still have to define what to do with all this.